website/docs/d/fabric_network.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "vra"
+page_title: "VMware vRealize Automation: fabric_network"
+description: |-
+  Provides a data lookup for vRA fabric networks.
+---
+
+# Data Source: fabric_network
+## Example Usages
+This is an example of how to lookup fabric networks.
+
+**Fabric network data source by Id:**
+
+```hcl
+# Lookup vRA fabric network using its Id
+data "fabric_network" "this" {
+  id = var.fabric_network_id
+}
```

Comment:
Currently, `vra_fabric_network` data source doesn't support lookup by id. This needs to be removed.